andela/mmdp_site
# MMDP-757 Automatically separate triple figures when entering values in the 'Beneficiary Count' field

**What does this PR do?**

Automatically separates triple figures when entering values in the 'Beneficiary Count' field.

**description of Task to be completed?**

The system automatically separates triple figures when entering values in the 'Beneficiary Count' field.

**How should this be manually tested?**

- `git fetch`
- `git checkout ft-separate-triple-figures-on-BeneficiaryCount-MMDP-757`
- `yarn dev`
- Open `http://localhost:1234` on your browser and navigate to `http://localhost:1234/stakeholder-directory`
- Edit or create a new stakeholder
- Triple figures on the beneficiary count field should be automatically separated

**What are the relevant Jira issues?**

[MMDP-757](https://viisaus.atlassian.net/browse/MMDP-757)

**Screenshots (if appropriate)**
<img width="1440" alt="Screenshot 2019-09-13 at 09 31 02" src="https://user-images.githubusercontent.com/29132159/64841940-4a3be600-d609-11e9-98bc-f8351a881e85.png">
